### PR TITLE
ssp2 should be 8-channel

### PR DESCRIPTION
**Pull request by zhigang-wu**

### CODE REVIEW

File: topology/sof-apl-tdf8532.m4 [hunk 1]
```diff
@@ -190,7 +190,7 @@ DAI_CONFIG(SSP, 2, 2, SSP2-Codec,
 	   SSP_CONFIG(DSP_B, SSP_CLOCK(mclk, 24576000, codec_mclk_in),
 		      SSP_CLOCK(bclk, 12288000, codec_slave),
 		      SSP_CLOCK(fsync, 48000, codec_slave),
-		      SSP_TDM(2, 32, 255, 255),
+		      SSP_TDM(8, 32, 255, 255),
```
Comment (keyonjie) [inline topology/sof-apl-tdf8532.m4]: how about the masks? do we need enable all 8 bits for all slots of Tx and Rx?

Comment (zhigang-wu) [reply topology/sof-apl-tdf8532.m4]: Sure, the TX and RX use the same clock and fs. When you set the RX with TDM8, you also should set the TX as the TDM8.

**Comment by zhigang-wu** [comment]
thank you, Pierre. I will reorder it.

**Comment by lgirdwood** [comment]
please also tell me which field you are reordering on ?

**Comment by zhigang-wu** [comment]
The DAI_ADD() and DAI_CONFIG() listed according on SSP port number, from SSP0 to SSP5

**Comment by zhigang-wu** [comment]
I update the commit (707d6d6) description already.
reorder according on the SSP port number.

**Pull request merged**